tests/issues/issue-1430.php

```diff
@@ -2789,9 +2789,9 @@
             'Device_Pointing_Method' => 'touchscreen',
             'Device_Code_Name' => 'iPhone',
             'Device_Brand_Name' => 'Apple',
-            'RenderingEngine_Name' => 'Blink',
```

Comment:
I did end up catching a pattern that was set as `Blink` even though it was an iOS pattern, that's why these tests were changed.

Reply:
It wasn't explicitly set as `Blink`, but was in a UA group that was set to `Blink` and didn't have a `WebKit` override. It has been moved to an iOS group now that is set to `WebKit` as the group's engine.